### ISSUE TITLE
Har file is messed by multiple rounds of waterfall charts

### ISSUE THREAD

**Issue by joychester** [sitespeedio/sitespeed.io#707]
I am running against firefox browser with 5 rounds, checked with har file, the first round is good, however, starting from second rounds, the har file is messed with multiple rounds of waterfall charts.

Screenshots:
<img width="973" alt="screen shot 2015-07-20 at 4 01 30 pm" src="https://cloud.githubusercontent.com/assets/235618/8771842/9f5bb690-2ef8-11e5-99e2-73a28a02eeb0.png">

Test sample:
sitespeed.io -u http://www.stubhub.com/performer/136034 -b firefox -n 5 -r ./sitespeed-result/Test_Page --suppressDomainFolder

Sitespeed version : 3.7.1

**Comment by soulgalore** [comment]
Hi @joychester ,
thanks for reporting! That looks wrong, I'll add a issue in Browsertime and reference it here and will have a look as soon as I have time.

Best
Peter

**Comment by soulgalore** [comment]
Yep, I tested know, the HAR-file is wrong, and it is wrong not depending on Firefox nor which site. Hmm, I'm pretty sure it hasnt always been, I'l try to hook up with @tobli and do a fix in Browsertime asap. Thanks again for reporting.

Best
Peter

**Comment by soulgalore** [comment]
I've fixed this in Browsertime but got a too bad internet connection to publish a new release. Will be there in aprox 24 hours. Thanks again for reporting!

**Comment by joychester** [comment]
Thanks for your quickly fix peter! Have a great vocation :)

**Comment by soulgalore** [comment]
thanks @joychester 
now t is released 3.7.2.
Best
Peter

**Comment by joychester** [comment]
Verified, it is fixed! :100:

**Comment by soulgalore** [comment]
perfect :)
